pkg/sdk/go/sdk.go

```diff
@@ -85,6 +89,11 @@ type User struct {
 	Password string                 `json:"password,omitempty"`
 	Metadata map[string]interface{} `json:"metadata,omitempty"`
 }
+type UserPage users.PageMetadata
+type UserPageMetadata struct {
+	Metadata map[string]interface{} `json:"metadata,omitempty"`
+	UserPage
+}
```

Comment:
Don't import Users service as a dependency to SDK. We are trying to keep SDK as simple and independent as possible, even at cost of copying structs from other services. Also, there is nothing wrong with using a single metadata page that joins all the metadata fields for all the entities in the SDK; unused fields will be simply ignored.

Reply:
I am reverting to using one PageMetadata which contains all the information for paging. For users, we will be interested in email while for things and channels we will be interested in the name
```go
type PageMetadata struct {
	Total    uint64
	Offset   uint64
	Limit    uint64
	Email    string
	Name     string
	Metadata map[string]interface{}
}
```